### PR TITLE
Nhd hr

### LINKED ISSUE #78: munge params into .nml-ready formats as part of `4_params_munge`
Running list of things we'll need for the glm3.nml file:

 - [x] `min_layer_thick` and `max_layer_thick` (in `&glm_setup`)
 - [x] `latitude` and `longitude` (in `&morphometry`)
 - [x] `bsn_len` and `bsn_wid` (in `&morphometry`)
 - [ ] ~`crest_elev` (in `&morphometry` or use default for now?)~
 - [x] `H` and `A` from hypsography (in `&morphometry` and relative to `crest_elev`)
 - [x] `cd` calculated from NLCD buffers (in `&meteorology`)
 - [ ] ~`Kw` or `Kw_file` (in `&light`)~ skipping this for now
 - [x] `meteo_fl` for driver file name (or x y cell indices)
 - [x] `lake_depth` from `H` and `crest_elev` (in `&init_profiles`)

### PR DESCRIPTION
**Pull request by jread-usgs**
fixes #78

### CODE REVIEW

**Comment by jread-usgs** [comment]
fixes #44 too

File: 4_params_munge/in/nlcd_class_heights.tsv [hunk 1]
```diff
@@ -0,0 +1,21 @@
+nlcd.class	hc.value	common.name
```
Comment (jzwart) [inline 4_params_munge/in/nlcd_class_heights.tsv]: what's `hc.value`?

File: 4_params_munge/src/munge_cd_from_nlcd.R [hunk 2]
```diff
@@ -0,0 +1,36 @@
+
+
+munge_cd_from_nlcd <- function(out_ind, nlcd_ind, nlcd_hc_file, areas_ind){
+  # need hc (canopy height) from nlcd class
+  min_wstr <- 0.0001
```
Comment (jzwart) [inline 4_params_munge/src/munge_cd_from_nlcd.R]: could you briefly define what each of these are in comments after the value?

File: 4_params_munge/in/nlcd_class_heights.tsv (hunk 1 above)
Comment (jzwart) [reply 4_params_munge/in/nlcd_class_heights.tsv]: ah canopy height, right? I read it in the code later on

File: 4_params_munge/src/munge_cd_from_nlcd.R [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  coef_wind_drag.ref <- 0.00140
+
+  nlcd_hc <- readr::read_tsv(nlcd_hc_file, col_types = 'idc') %>%
+    dplyr::select(nlcd_class = nlcd.class, hc_value = hc.value) %>%
+    rbind(data.frame(nlcd_class = 0, hc_value = 0)) # great lakes seem to have "0"?
```
Comment (jzwart) [inline 4_params_munge/src/munge_cd_from_nlcd.R]: Great Lakes have a 0 NLCD class?

File: 4_params_munge/src/munge_cd_from_nlcd.R [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  blank_nlcd_frac_classes <- nlcd_frac_classes %>% dplyr::select(-id)
+
+  lake_hc <- nlcd_frac_classes %>% # this following line gets the name of the column for the max canopy fraction in each row
+    mutate(max_class_name = colnames(blank_nlcd_frac_classes )[max.col(blank_nlcd_frac_classes ,ties.method="first")]) %>%
+    rowwise() %>%
+    mutate(nlcd_class = as.integer(strsplit(max_class_name,'[.]')[[1]][2])) %>%
+    dplyr::select(site_id = id, nlcd_class) %>%
+    left_join(nlcd_hc) %>%
+    left_join(lake_areas) %>%
+    mutate(Xt = 50 * hc_value,
```
Comment (jzwart) [inline 4_params_munge/src/munge_cd_from_nlcd.R]: where is this equation coming from?

**Pull request merged**